content/en/docs/userguide/install_waypoint.md

```diff
@@ -114,40 +106,33 @@ sleep-5577c64d7c-n7rxp                    1/1     Running   0          30m
 ```
 
 
-#### Install waypoint in namespace granularity:
-
-```bash
-[root@ ~]#  istioctl x waypoint apply -n default --name default-ns-waypoint
-waypoint default/default-ns-waypoint applied
-
-[root@ ~]#  kubectl label namespace default istio.io/use-waypoint=default-ns-waypoint
-namespace/default labeled
-```
+#### Configure waypoint for a specific namespace:
 
-***NOTE: Also need to replace the original image of waypoint with the Kmesh customized image.***
+Deploy a waypoint for the `default` with default name `waypoint`. By specifying `--enroll-namespace`, the namespace will be labeled with `istio.io/use-waypoint=waypoint`
 
 ```bash
-[root@ ~]# kubectl annotate gateway default-ns-waypoint sidecar.istio.io/proxyImage=ghcr.io/kmesh-net/waypoint:latest
+[root@ ~]# kmeshctl waypoint apply -n default --enroll-namespace
+
+waypoint default/default-ns-waypoint applied
 ```
+#### Configure waypoint for a specific pod:
 
-Then any requests from any pods using the Kmesh, to any service running in `default` namespace, will be routed through that waypoint for L7 processing and policy enforcement.
+Deploy a waypoint called reviews-v2-pod-waypoint for the `reviews-v2-5979c6fc9c-72bst` pod.
 
-#### Install waypoint in pod granularity:
 
 ```bash
-[root@ ~]# istioctl x waypoint apply -n default --name reviews-v2-pod-waypoint --for workload
+[root@ ~]# kmeshctl waypoint apply -n default --name reviews-v2-pod-waypoint --for workload
 waypoint default/reviews-v2-pod-waypoint applied
 # Label the `reviews-v2` pod to use `reviews-v2-pod-waypoint` waypoint.
-[root@ ~]# kubectl label pod -l version=v2,app=reviews istio.io/use-waypoint=reviews-v2-pod-waypoint
+[root@ ~]# kubectl label pod reviews-v2-5979c6fc9c-72bst istio.io/use-waypoint=reviews-v2-pod-waypoint
 pod/reviews-v2-5b667bcbf8-spnnh labeled
 ```
 
-***NOTE: Also need to replace the original image of waypoint with the Kmesh customized image.***
+***NOTE: If you need to replace the original image of waypoint with the Kmesh customized image.***
```

Comment:
No more image replacement operations are required.

Reply:
nope, currently if user build kmeshctl locally, it could generate yaml with  `:vv0.0.0-master` image